packages/conf-brotli/conf-brotli.0.0.1/url

```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/fxfactorial/conf-brotli/archive/v0.0.1.tar.gz"
```

Comment:
There's no need to maintain your [own repo](https://github.com/fxfactorial/conf-brotli) for `conf-brotli` the `conf-*` packages are maintained in the `opam-repository` itself simply PR the `opam-repository` with the files.